docs/api-docs/payments/payments-api-overview.md

```diff
@@ -35,22 +35,37 @@ You can process payments charged to either of two main forms of payment: new pay
 > * The API flow does not currently support hosted, offsite, or wallet-type providers, such as Amazon Pay.
 
 ## Stored cards and PayPal accounts
-There are three steps to using a stored card or PayPal account to make a payment.
-
-1. [Get Payment Methods](/api-reference/store-management/payment-processing/accepted-methods/paymentsmethodsget)
-2. [Create Access Token](/api-reference/store-management/payment-processing/access-tokens/paymentsaccesstokenspost)
-3. [Process Payment](/api-reference/store-management/payment-processing/process-payment/paymentspost)
-
-To use stored cards with the Payments API or the Checkout SDK, make sure you enable stored cards in the store's control panel. To enable stored credit cards on your storefront, navigate to **Store Setup › Payments**, and click the tab for your payment gateway. Toggle the switch to enable Stored Credit Cards and click **Save**. For more on enabling stored payment methods, see [Enabling Stored Payment Methods](https://support.bigcommerce.com/s/article/Enabling-Stored-Credit-Cards).
 
 <!-- theme: info -->
 > #### Requirements for stored cards
 > * Your store must be on a Plus plan or higher.
 > * Your store must use Optimized One-Page Checkout.
 > * Your store must use a compatible payment gateway.
 
+<!-- theme: info -->
```

Comment:
Do these need to be callouts? They wind up being so big that it looks like it could just be part of the doc 
![image](https://user-images.githubusercontent.com/29524703/160712746-f6afae40-02cf-42ac-a730-0c4d79196098.png)